autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+#_preseed_V1
+#### Contents of the preconfiguration file (for pardus)
+
+d-i debconf/language string en_US:en
+d-i debian-installer/country string US
+d-i debian-installer/fallbacklocale string en_US
+d-i debian-installer/language string en_US:en
+d-i languagechooser/locale string en
+
+### Localization
+# Preseeding only locale sets language, country and locale.
+d-i debian-installer/locale string en_US
+
+# Keyboard selection.
+d-i keyboard-configuration/xkb-keymap select us
+d-i keyboard-configuration/variant select English (US)
+
+### Network configuration
+# Disable network configuration entirely. This is useful for cdrom
+# installations on non-networked devices where the network questions,
+# warning and long timeouts are a nuisance.
+d-i netcfg/enable boolean true
+
+# netcfg will choose an interface that has link if possible. This makes it
+# skip displaying a list if there is more than one interface.
+d-i netcfg/choose_interface select auto
+d-i netcfg/use_dhcp string true
+
+# To set a different link detection timeout (default is 3 seconds).
+# Values are interpreted as seconds.
+d-i netcfg/link_wait_timeout string 10
+
+# If you have a slow dhcp server and the installer times out waiting for
+# it, this might be useful.
+d-i netcfg/dhcp_timeout string 60
+d-i netcfg/dhcpv6_timeout string 60
```

Comment:
There might be no DHCP server for IPv6. I think it is not necessary to set dhcpv6_timeout

Reply:
Change it . Thanks